cmd/gctcli/commands.go

```diff
@@ -5060,3 +5060,202 @@ func shutdown(c *cli.Context) error {
 	jsonOutput(result)
 	return nil
 }
+
+var getMarginRatesHistoryCommand = &cli.Command{
+	Name:      "getmarginrateshistory",
+	Usage:     "returns margin lending/borrow rates for a period",
+	ArgsUsage: "<exchange> <asset> <currency> <start> <end> <getpredictedrate> <getlendingpayments> <getborrowrates> <getborrowcosts> <includeallrates>",
+	Action:    getMarginRatesHistory,
+	Flags: []cli.Flag{
+		&cli.StringFlag{
+			Name:    "exchange",
+			Aliases: []string{"e"},
+			Usage:   "the exchange to retrieve futures positions from",
+		},
+		&cli.StringFlag{
+			Name:    "asset",
+			Aliases: []string{"a"},
+			Usage:   "the asset type of the currency pair, must be a futures type",
+		},
+		&cli.StringFlag{
+			Name:    "currency",
+			Aliases: []string{"c"},
+			Usage:   "must be an enabled currency",
+		},
+		&cli.StringFlag{
+			Name:        "start",
+			Aliases:     []string{"sd"},
+			Usage:       "<start>",
+			Value:       time.Now().AddDate(0, -1, 0).Truncate(time.Hour).Format(common.SimpleTimeFormat),
+			Destination: &startTime,
+		},
+		&cli.StringFlag{
+			Name:        "end",
+			Aliases:     []string{"ed"},
+			Usage:       "<end>",
+			Value:       time.Now().Format(common.SimpleTimeFormat),
+			Destination: &endTime,
+		},
+		&cli.BoolFlag{
+			Name:    "getpredictedrate",
+			Aliases: []string{"p"},
+			Usage:   "include the predicted upcoming rate in the response",
+		},
+		&cli.BoolFlag{
+			Name:    "getlendingpayments",
```

Comment:
```
 go run . getmarginrateshistory -e='ftx' -a='spot' -c='btc' -br -bc --lp
```
 "sum_lending_payments": "0.10386594827889",
 "sum_lending_size": "2.46127894827889", 

The lending size of the account is 0.00763200 the above values seem to be exaggerated somewhat.

Doing verbose: 
![image](https://user-images.githubusercontent.com/9391715/174940120-b3b0a292-5fdc-4d09-82df-5e17f05a8d70.png)

The size randomly jumps up. 🤷

Reply:
Definitely no copy pasting was involved in any errors 👀  https://github.com/thrasher-corp/gocryptotrader/pull/981/files#diff-e7811fdafb5e931926e53a40cf51d3205cfc00947aa56c5e0bac7635e88ad6f7R1828